### ISSUE TITLE
Rename the `-p` flag?

### ISSUE THREAD

**Issue by nabuskey** [cnoe-io/idpbuilder#264]
Do we want to rename the `--package-dir` flag? Right now, it expects a directory that contain ArgoCD application files but this behaviour will likely change soonish. If that happens, we should consider changing this flag's name. 

We should ensure that `--package-dir` flag works going forward for backward compatibility purposes though.

**Comment by cmoulliard** [comment]
> Do we want to rename the `--package-dir` flag?

I'm in favor to rename it in order to been able to create an IDP cluster using packages coming from different locations. Of course, that will imply that we prefix each package with a URI even if we will upload the package resources using a folder/dir 

```
--package files:///a/b/c
--package https://
--package ssh://
--package git://
-- package ftp:///
```

**Comment by nimakaviani** [comment]
yeah should be an easy lift to add the new flag as an alternative and plan to deprecate this in a v0.6 release.

**Comment by squidboylan** [comment]
Is there anything left to do before closing this? Looks like this is fixed

**Comment by nabuskey** [comment]
Agreed. Let's close it.
